tests/test_custom_checkers/test_pycodestyle_checker.py

```diff
@@ -169,18 +185,122 @@ def test_no_error_e262(self) -> None:
         with self.assertNoMessages():
             self.checker.process_module(mod)
 
-    def test_error_e265(self) -> None:
-        """Test that PEP8 error E265 (block comment should start with '# ') triggers"""
-        mod = MANAGER.ast_from_file(DIR_PATH + "e265_error.py")
-        args = ("E265", "line 1, column 0: block comment should start with '# '")
+    def test_error_e115(self) -> None:
+        """Test that PEP8 error E115 (Expected an indented block) is triggered"""
+        mod = MANAGER.ast_from_file(DIR_PATH + "e115_error.py")
         with self.assertAddsMessages(
-            pylint.testutils.MessageTest(msg_id="pep8-errors", line=1, node=None, args=args),
-            ignore_position=True,
+            pylint.testutils.MessageTest(
+                msg_id="pep8-errors",
+                line=2,
+                args=("E115", "line 2, column 0: expected an indented block (comment)"),
+            )
         ):
             self.checker.process_module(mod)
 
-    def test_no_error_e265(self) -> None:
```

Comment:
@Raine-Yang-UofT, I think you may have forgotten to pull before the most recent commit you made. Your PR essentially deletes my PR, since my work had not been merged when you created your branch. 

I believe if you do 
```git checkout your-branch-name
git pull upstream master
*resolve merge conflicts*
git push origin your-branch-name
```
that should fix this problem.

Reply:
Alternatively, Professor @david-yz-liu may prefer to resolve the conflicts on his end? I would imagine it is better to resolve such conflicts before merging with `master` though.